Admin/CategoryAdmin.php

```diff
@@ -47,11 +46,9 @@ protected function configureFormFields(FormMapper $formMapper)
             if ($this->getSubject()->getParent() !== null || $this->getSubject()->getId() === null) { // root category cannot have a parent
                 $formMapper
                   ->add('parent', 'sonata_category_selector', array(
-                      'category' => $this->getSubject() ?: null,
                       'model_manager' => $this->getModelManager(),
                       'class' => $this->getClass(),
                       'required' => true,
-                      'context' => $this->getSubject()->getContext(),
```

Comment:
[Here is how you can fix this](https://github.com/sonata-project/SonataAdminBundle/blob/3.x/Form/Type/AdminType.php#L108)